src/main/java/uk/gov/hmcts/reform/et/syaapi/service/StoredApplicationService.java

```diff
@@ -86,4 +97,114 @@ private void sendEmailForApplication(CaseDetails finalCaseDetails, CaseData case
     }
+
+    /**
+     * Submit stored claimant response to Tribunal response.
+     *
+     * @param authorization - authorization
+     * @param request - response from the claimant
+     * @return the associated {@link CaseDetails} for the ID provided in request
+     */
+    public CaseDetails submitRespondToApplication(String authorization,
+                                                  UpdateStoredRespondToApplicationRequest request) {
+        String caseId = request.getCaseId();
+        String caseTypeId = request.getCaseTypeId();
+
+        StartEventResponse startEventResponse = caseService.startUpdate(
+            authorization,
+            caseId,
+            caseTypeId,
+            CaseEvent.CLAIMANT_TSE_RESPOND
+        );
+
+        CaseData caseData = EmployeeObjectMapper
+            .mapRequestCaseDataToCaseData(startEventResponse.getCaseDetails().getData());
+
+        // Get selected GenericTseApplicationTypeItem
+        GenericTseApplicationTypeItem appToModify = TseApplicationHelper.getSelectedApplication(
+            caseData.getGenericTseApplicationCollection(), request.getApplicationId()
+        );
+        if (appToModify == null) {
+            throw new IllegalArgumentException(APP_ID_INCORRECT);
+        }
+
+        // Get selected TseRespondTypeItem
+        TseRespondTypeItem responseToModify = TseApplicationHelper.getResponseInSelectedApplication(
+            appToModify.getValue().getRespondCollection(), request.getRespondId()
+        );
+        if (responseToModify == null) {
+            throw new IllegalArgumentException(RESPOND_ID_INCORRECT);
+        }
+
+        // Update response details and application status
+        updateResponseForSubmitStored(responseToModify, caseData, appToModify);
+
+        // Update pdf
+        createAndAddPdfOfResponse(authorization, request, caseData, appToModify.getValue(),
+            responseToModify.getValue());
+
+        // Send confirmation email
+        sendEmailForRespondToApplication(caseData, caseId, appToModify, request.isRespondingToRequestOrOrder());
+
+        return caseService.submitUpdate(
+            authorization, caseId, caseDetailsConverter.caseDataContent(startEventResponse, caseData), caseTypeId);
+    }
+
+    private static void updateResponseForSubmitStored(TseRespondTypeItem responseToModify, CaseData caseData,
+                                                      GenericTseApplicationTypeItem appToModify) {
+        TseRespondType tseRespondType = responseToModify.getValue();
+        if (tseRespondType.getSupportingMaterial() != null) {
+            DocumentTypeItem documentTypeItem =
+                (DocumentTypeItem) tseRespondType.getSupportingMaterial().get(0);
+            if (caseData.getDocumentCollection() == null) {
+                caseData.setDocumentCollection(new ArrayList<>());
+            }
+            caseData.getDocumentCollection().add(documentTypeItem);
```

Comment:
Above if condition can be checked better using CollectionUtils methods for null/empty list.